src/applications/benefit-eligibility-questionnaire/pages/militaryService.js

```diff
@@ -28,6 +41,16 @@ export default {
         type: 'string',
         enum: ['Yes', 'No'],
       },
+      expectedSeparation: {
+        type: 'string',
+        enum: [
+          'Within the next 3 months',
+          'More than 3 months but less than 6 months',
+          'Within more than 6 months but less than 1 year',
```

Comment:
Should this sentence instead be "More than 6 months but less than 1 year"?

Reply:
Good catch! Thanks